morphia/src/main/java/org/mongodb/morphia/DatastoreImpl.java

```diff
@@ -702,44 +703,50 @@ public void setQueryFactory(final QueryFactory queryFactory) {
 
     @Override
     public <T> Iterable<Key<T>> save(final Iterable<T> entities) {
-        if (entities == null) {
-            return new ArrayList<Key<T>>();
-        }
         Iterator<T> iterator = entities.iterator();
-        if (!iterator.hasNext()) {
-            return new ArrayList<Key<T>>();
-        }
-        return save(entities, getWriteConcern(iterator.next()));
+        return !iterator.hasNext()
+               ? Collections.<Key<T>>emptyList()
+               : save(entities, getWriteConcern(iterator.next()));
     }
 
     @Override
     public <T> Iterable<Key<T>> save(final Iterable<T> entities, final WriteConcern wc) {
+        return save(entities, new InsertOptions().writeConcern(wc));
+    }
+
+    @Override
+    public <T> Iterable<Key<T>> save(final Iterable<T> entities, final InsertOptions options) {
         final List<Key<T>> savedKeys = new ArrayList<Key<T>>();
         for (final T ent : entities) {
-            savedKeys.add(save(ent, wc));
+            savedKeys.add(save(ent, options));
         }
         return savedKeys;
 
     }
 
     @Override
+    @Deprecated
     public <T> Iterable<Key<T>> save(final T... entities) {
-        final List<Key<T>> savedKeys = new ArrayList<Key<T>>();
-        for (final T ent : entities) {
-            savedKeys.add(save(ent));
-        }
-        return savedKeys;
+        return save(asList(entities), new InsertOptions());
```

Comment:
This looks like a behavioral change.  In the previous version, each entity is saved with the default write concern for the entity by calling DatastoreImpl#getWriteConcern.  In this version the default write concern (null) of InsertOptions will be used.

I suggest auditing all usage of InsertOptions to ensure that the default write concern is always applied.